### ISSUE TITLE
[dev.icinga.com #8534] support livestatus wrapped_json format

### ISSUE THREAD

**Issue by icinga-migration** [Icinga/icinga2#2668]
This issue has been migrated from Redmine: https://dev.icinga.com/issues/8534

**Created by sni on 2015-02-25 11:37:59 +00:00**

Assignee: _(none)_
Status: _Rejected_  (closed on _2016-02-25 00:27:15 +00:00_)
Target Version: _(none)_
Last Update: _2016-02-25 00:27:15 +00:00 (in Redmine)_

    Backport?: Not yet backported
    Include in Changelog: 1

---

It would be nice to support the wrapped\_json outputformat like naemons livestatus does:

    echo -e "GET services\nColumns: host_name description state\nOutputFormat: wrapped_json\nLimit: 1\nOffset: 5\nSort: host_name asc\nSort: description asc\n" | unixcat...
    {"data":[["hostname","service",0]],"total_count":108}

There is some description available here:
http://www.naemon.org/documentation/usersguide/livestatus.html#extented\_functionality

Basically it allows you to sort querys and retrieve only a subset of the result while also having the number of the total result set available. This would make livestatus querys way more efficient.
